client/src/app/videos/+video-watch/video-watch.component.ts

```diff
@@ -140,6 +141,11 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
       }, undefined, this.i18n('Subscribe to the account'))
     ]
     if (this.isUserLoggedIn()) this.hotkeysService.add(this.hotkeys)
+
+    if (this.user.pipPlayer) {
+      fromEvent(window, 'scroll')
```

Comment:
add debounce or throttle (you have an example in the infinite scroll directive)

Reply:
do you think a debounce would feel nice?
I think this might feel strange if it takes a time until the change appears, therefor I removed the debounce time.